### ISSUE TITLE
Fatal error: Uncaught TypeError: Argument 1 passed to Lcobucci\JWT\Signer\Ecdsa\KeyParse

### ISSUE THREAD

**Issue by craigcarnell** [lcobucci/jwt#106]
```
Fatal error: Uncaught TypeError: Argument 1 passed to Lcobucci\JWT\Signer\Ecdsa\KeyParser::__construct() must be an instance of Mdanter\Ecc\Math\MathAdapterInterface, instance of Mdanter\Ecc\Math\GmpMath given, called in vendor/lcobucci/jwt/src/Signer/Ecdsa.php on line 50 and defined in vendor/lcobucci/jwt/src/Signer/Ecdsa/KeyParser.php:43
```

**Comment by lcobucci** [comment]
@craigcarnell some questions for you 😄:

- what version of `lcobucci/jwt` and `mdanter/ecc` are you using? 
- how did you installed the it?

**Comment by craigcarnell** [comment]
@lcobucci jwt is master, mdanter/ecc is 0.4.0

It looks like in 0.4.0 they removed the class. If I use the last release 0.3.1 it works

**Comment by lcobucci** [comment]
Thanks!

I'll fix the `0.3.1` on our stable version (3.1) and schedule the compatibility with `0.4.0` to our development version (4.0).

Please make sure you don't use our unstable/development version on production because it will change without any notice.
